src/structs/database.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+import { Client, BucketItem } from "minio";
```

Comment:
`BucketItem` is not used, you can remove the import

Reply:
Fixed in 74df2b3